dex/src/main/scala/com/wavesplatform/dex/api/http/entities/HttpSystemStatus.scala

```diff
@@ -0,0 +1,19 @@
+package com.wavesplatform.dex.api.http.entities
+
+import com.wavesplatform.dex.app.MatcherStatus
+import com.wavesplatform.dex.grpc.integration.clients.combined.CombinedStream
+import io.swagger.annotations.ApiModelProperty
+import play.api.libs.json.{Format, Json}
+
+case class HttpSystemStatus(
+  @ApiModelProperty() service: String,
+  @ApiModelProperty() blockchain: String
```

Comment:
It'd better to have `MatcherStatus` and `CombinedStream.Status` here, because:
1. We will use these fields in tests later.
2. It is easy to make a mistake in string, a compiler won't help. But it helps in a case of types

Reply:
done